testplan/testing/multitest/entries/base.py

```diff
@@ -342,3 +328,17 @@ def __init__(self, filepath, description):
             filesize=self.filesize,
             ext=ext)
         super(Attachment, self).__init__(description=description)
+
+
+class MatPlot(Attachment):
+    """Display a MatPlotLib graph in the report."""
+    def __init__(self, pyplot, image_file_path, width=2, height=2,
+                 description=None):
+        self.width = float(width)
+        self.height = float(height)
+        self.image_file_path = image_file_path
```

Comment:
Nice! Just one thing: the Attachment entry already stores the original filepath via the `source_path` attribute, so I think that also storing it in the `image_file_path` attribute is redundant.